bin/poll.py

```diff
@@ -23,38 +23,50 @@
 
 BUILDS_ENDPOINT = 'https://api.buildkite.com/v2/organizations/clima/builds'
 
-# we pick an hour timedelta for the build window, 
+# we pick an hour timedelta for the build window,
 # but it just needs to be more than that cron poll interval
 def hour_ago_utc():
     return (datetime.utcnow() - timedelta(hours=1)).replace(microsecond=0).isoformat() + 'Z'
 
 
 def all_started_builds():
-    state_query = '&state[]=scheduled&state[]=running'
-    created_from_query = '&created_from={0}'.format(hour_ago_utc())
+    since = hour_ago_utc()
     npage, builds = 1, []
     while True:
-        page_query = '?page={0}&per_page=100'.format(npage)
-        req_url = BUILDS_ENDPOINT + page_query + state_query + created_from_query
         resp = requests.get(
-            req_url, headers={'Authorization': 'Bearer ' + BUILDKITE_API_TOKEN}
+            BUILDS_ENDPOINT,
+            params = {
+                'page' : npage,
+                'per_page' : 100,
+                'state[]' : ['scheduled', 'running'],
+                'created_from' : since
+            },
+            headers = {
+                'Authorization': 'Bearer ' + BUILDKITE_API_TOKEN
+            }
         ).json()
         if not len(resp):
             break
         builds.extend(resp)
         npage += 1
-    return builds 
+    return builds
+
 
- 
 def all_canceled_builds():
-    state_query = '&state[]=canceling&state[]=canceled'
-    finished_from_query = '&finished_from={0}'.format(hour_ago_utc())
+    since = hour_ago_utc()
     npage, builds = 1, []
     while True:
-        page_query = '?page={0}per_page=100'.format(npage)
-        req_url = BUILDS_ENDPOINT + page_query + state_query + finished_from_query
         resp = requests.get(
-            req_url, headers={'Authorization': 'Bearer ' + BUILDKITE_API_TOKEN}
+            BUILDS_ENDPOINT,
+            params = {
+                'page' : npage,
+                'per_page' : 100,
+                'state[]' : ['canceling', 'canceled'],
+                'created_from' : since
```

Comment:
@simonbyrne I think this should be finished_from.

Reply:
Yeah, I realized that. I can fix it tonight.